### PR TITLE
removing advanced options for DIDs

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: src/features/didManagement/CreateNewDIDScreen.js [hunk 1]
```diff
@@ -21,7 +21,6 @@ import {
   VStack,
 } from 'native-base';
 import {translate} from '../../locales';
-import {DIDAdvancedOptions} from './components/DIDAdvancedOptions';
```
Comment (mike-parkhill) [inline src/features/didManagement/CreateNewDIDScreen.js:24]: should this component file be deleted too?

**Review by mike-parkhill — COMMENTED** [review]
Will the default be set properly with this block gone?

I imagine there might need to be matching updates in the the automated UI tests

**Comment by maycon-mello** [comment]
> Will the default be set properly with this block gone?
> 
> I imagine there might need to be matching updates in the the automated UI tests

Yes, this block was optional

File: src/features/didManagement/CreateNewDIDScreen.js (hunk 1 above)
Comment (maycon-mello) [reply src/features/didManagement/CreateNewDIDScreen.js:24]: The component was deleted in the latest commit

**Pull request merged**